apps/cowswap-frontend/src/modules/application/containers/App/styled.ts

```diff
@@ -44,6 +50,13 @@ export const BodyWrapper = styled.div<{ customTheme?: CowSwapTheme }>`
 
       if (customTheme === ('darkHalloween' as CowSwapTheme)) {
         backgroundImage = `url(${IMAGE_BACKGROUND_DARK_HALLOWEEN})`
+      } else if (
+        customTheme === ('darkChristmas' as CowSwapTheme) ||
```

Comment:
Nitpick: `as CowSwapTheme` is redundant here

Reply:
Might be just: 
`['darkChristmas', 'lightChristmas'].includes(customTheme)`

Reply:
Refactored the christmas theme detection 57f809bafd5b9d2c939287e42a3d55bdd3ecaaa0